taskcluster/requirements.in

```diff
@@ -1,7 +0,0 @@
-# For instructions on managing dependencies, see:
-# https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html
-
-arrow
-pyyaml
-taskcluster
```

Comment:
While taskgraph is baked in, these other packages are not. So we still need the requirements files for those.

Reply:
It doesn't appear they're actually used. At least, the decision task runs fine without them!